x-pack/test/security_solution_cypress/es_archives/threat_indicator/mappings.json

```diff
@@ -2,29 +2,821 @@
+      "_meta": {
+        "beat": "filebeat",
+        "version": "7.12.0"
+      },
+      "date_detection": false,
+      "dynamic_templates": [
+        {
+          "labels": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "match_mapping_type": "string",
+            "path_match": "labels.*"
+          }
+        },
+        {
+          "container.labels": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "match_mapping_type": "string",
+            "path_match": "container.labels.*"
+          }
+        },
+        {
+          "fields": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "match_mapping_type": "string",
+            "path_match": "fields.*"
+          }
+        },
+        {
+          "docker.container.labels": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "match_mapping_type": "string",
+            "path_match": "docker.container.labels.*"
+          }
+        },
+        {
+          "kubernetes.labels.*": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "kubernetes.labels.*"
+          }
+        },
+        {
+          "kubernetes.annotations.*": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "kubernetes.annotations.*"
+          }
+        },
+        {
+          "kubernetes.service.selectors.*": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "kubernetes.service.selectors.*"
+          }
+        },
+        {
+          "docker.attrs": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "match_mapping_type": "string",
+            "path_match": "docker.attrs.*"
+          }
+        },
+        {
+          "azure.activitylogs.identity.claims.*": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "azure.activitylogs.identity.claims.*"
+          }
+        },
+        {
+          "azure.platformlogs.properties.*": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "path_match": "azure.platformlogs.properties.*"
+          }
+        },
+        {
+          "kibana.log.meta": {
+            "mapping": {
+              "type": "keyword"
+            },
+            "match_mapping_type": "string",
+            "path_match": "kibana.log.meta.*"
+          }
+        },
+        {
+          "strings_as_keyword": {
+            "mapping": {
+              "ignore_above": 1024,
+              "type": "keyword"
+            },
+            "match_mapping_type": "string"
+          }
+        }
+      ],
       "properties": {
         "@timestamp": {
           "type": "date"
         },
         "agent": {
           "properties": {
+            "build": {
```

Comment:
Do we want to trim this down further to _only_ the fields that we're using in the accompanying `data.json` ? I thought that was the consensus but maybe I misunderstood.